AutoCollection/Performance.ts

```diff
@@ -151,18 +130,18 @@ class AutoCollectPerformance {
                 totalIrq += irq;
 
                 var total = (user + sys + nice + idle + irq) || 1; // don"t let this be 0 since it is a divisor
-                this._client.trackMetric({name: name + "user", value: (user / total) * 100});
+                // this._client.trackMetric({name: name + "user", value: (user / total) * 100});
             }
 
-            var combinedName = "% total cpu";
+            // var combinedName = "% total cpu";
```

Comment:
Minor: I think we can remove unused/commented code and keep it in history if we need to enable it back.

Reply:
Fair point, will remove